### PR TITLE
Added firebase and firebaseConfig.js

### PR DESCRIPTION
**Pull request by OG-Jons**
#5

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    const firebaseApp = firebase.apps[0];
+    console.log(firebaseApp)
```
Comment (Nikzz3) [inline src/App.js:7]: remove console logs

Comment (OG-Jons) [reply src/App.js:7]: hans gmacht als biispil, sodass mer weiss, wie mer es innem komponente verwende cha

**Review by GcZuRi1886 — COMMENTED** [review]
console.log ?? xD

Ups ha din Comment ned gseh

File: src/App.js (hunk 1 above)
Comment (Nikzz3) [reply src/App.js:7]: Easy

**Review by Nikzz3 — APPROVED** [review]

**Review by Sl1van — APPROVED** [review]
Seems all good

**Review by GcZuRi1886 — APPROVED** [review]

**Pull request merged**